src/mode/logtalk_highlight_rules.js

```diff
@@ -86,7 +74,7 @@ var LogtalkHighlightRules = function() {
          { token: 'support.function.control.logtalk',
            regex: '\\b(?:true|fa(?:il|lse)|repeat|(?:instantiation|system)_error)\\b(?![-!(^~])' },
          { token: 'support.function.control.logtalk',
-           regex: '\\b((?:type|domain|existence|permission|representation|evaluation|resource|syntax)_error)(?=[(])' },
+           regex: '\\b((?:uninstantiation|type|domain|existence|permission|representation|evaluation|resource|syntax)_error)(?=[(])' },
```

Comment:
do we need here `instantiation_error` as well? [Ref](https://logtalk.org/manuals/refman/methods/error_handling_methods.html).

Reply:
No. `instantiation_error/0` is above that line as it doesn't have any arguments unlike e.g. `uninstantiation_error/1` that have one argument (and thus `uninstantiation_error` must be followed by a `(` character; no spaces between allowed).